doc/classes/Camera2D.xml

```diff
@@ -142,9 +145,17 @@
 		<member name="limit_bottom" type="int" setter="set_limit" getter="get_limit" default="10000000">
 			Bottom scroll limit in pixels. The camera stops moving when reaching this value, but [member offset] can push the view past the limit.
 		</member>
+		<member name="limit_disabled" type="bool" setter="set_limit_disabled" getter="is_limit_disabled" default="false">
+			If [code]true[/code], the limits will be disabled, which means that the camera can focus anywhere. In this case, the four [code]limit_*[/code] properties will not work.
+		</member>
 		<member name="limit_left" type="int" setter="set_limit" getter="get_limit" default="-10000000">
 			Left scroll limit in pixels. The camera stops moving when reaching this value, but [member offset] can push the view past the limit.
 		</member>
+		<member name="limit_rect_to_viewport" type="Callable" setter="" getter="get_limit_rect_to_viewport" default="Callable()">
+			A tool button that snaps the limits to the viewport.
+			In the editor, pressing the button will make [member limit_left] and [member limit_top] become the global position of the camera, and [member limit_right] and [member limit_bottom] become the global position plus the size of the viewport.
+			[b]Readonly:[/b] Since this is a tool button, it is read-only and trying to set the value of this property will trigger an error.
+		</member>
```

Comment:
I suspect we don't want to add/bind such button as a property with `PROPERTY_HINT_TOOL_BUTTON`... Probably should be instead added using an EditorInspectorPlugin [like e.g. `Edit Region` button for Sprite2D etc.](https://github.com/godotengine/godot/blob/d19147e09a13a7d6df44076f15893a5d1108fbb0/editor/plugins/texture_region_editor_plugin.cpp#L1282-L1311)? :thinking: cc @KoBeWi

Reply:
Yeah inspector plugin would avoid adding a property. The property is not needed for anything.